packages/react-devtools-scheduling-profiler/src/view-base/VerticalScrollView.js

```diff
@@ -165,6 +165,7 @@ export class VerticalScrollView extends View {
 
   _handleMouseDown(interaction: MouseDownInteraction) {
     if (rectContainsPoint(interaction.payload.location, this.frame)) {
+console.log('VerticalScrollView()\n  location:', interaction.payload.location, '\n  frame:', JSON.stringify(this.frame).replace('"', ''), '\n  visibleArea:', JSON.stringify(this.visibleArea).replace('"', ''));
```

Comment:
🤪

Reply:
Looks like this was an older version of the file? I don't think there's any console logs committed in this branch now (though it looks like I accidentally pushed a STASHING commit I didn't mean to early on 🤪